### PR TITLE
Fixed a charmap warning with the new RGBDS

### PR DESCRIPTION
**Pull request by coltongit**
Fixed the charmap warning issue as per https://github.com/aaaaaa123456789/pokecrystal16/commit/bb7b22e0dfddaa4def206982e5e151fa284a56ee

### CODE REVIEW

File: mobile/mobile_46.asm [hunk 1]
```diff
@@ -1484,7 +1484,8 @@ Function118b9a:
 	and a
 	ret
 
-	charmap " ", $20 ; revert to ascii
+	pushc
+	setcharmap ascii ; revert to ascii
```
Comment (mid-kid) [inline mobile/mobile_46.asm]: This use of setcharmap should end after the URLs.

File: mobile/mobile_46.asm [hunk 2]
```diff
@@ -2927,6 +2928,7 @@ Unknown_1196b8:
 	db "Sat"
 	db "Sun"
 
+popc
```
Comment (mid-kid) [inline mobile/mobile_46.asm]: You should wrap the weekdays here in a setcharmap instead of covering this entire area.

**Comment by mid-kid** [comment]
It also seems the travis build has failed. Run `make compare` before submitting your PRs to make sure the ROM still matches, and run a binary diff to see what you accidentally changed.

File: mobile/mobile_46.asm (hunk 1 above)
Comment (coltongit) [reply mobile/mobile_46.asm]: Resolved in latest commit.

File: mobile/mobile_46.asm (hunk 2 above)
Comment (coltongit) [reply mobile/mobile_46.asm]: Resolved in latest commit.

**Pull request closed without merging**